src/AlertBox/AlertBox.less

```diff
@@ -10,13 +10,15 @@
 }
 
 .AlertBox--title {
-  display: inline;
-  .margin--left--2xs;
+  .margin--left--xs;
   .text--semi-bold;
+  flex-grow: 1;
```

Comment:
This `flex-grow` makes sure that the close button sits at the right end of the header. (This wasn't necessary earlier because we weren't using flex styling for the header.)